datadog_lambda/trigger.py

```diff
@@ -0,0 +1,205 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019 Datadog, Inc.
+
+import base64
+import gzip
+import json
+from io import BytesIO, BufferedReader
+
+
+EVENT_SOURCES = [
+    "aws:dynamodb",
+    "aws:kinesis",
+    "aws:s3",
+    "aws:sns",
+    "aws:sqs",
+]
+
+GOVCLOUD_REGIONS = ["us-gov-east-1", "us-gov-west-1"]
```

Comment:
Any chance you can get the partition directly from the `event` or `context` object? Otherwise it's guaranteed that you would have to update these lists in the future. The worst case, can you use the region prefix, say `us-gov-` or `cn-` to determine partition?

Reply:
Yeah that's a really good point. The `event` object may not have the partition but we parse out the `region` from the Lambda `context` so it will probably work best to check the region starts with either of the partitions.